parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -1196,3 +1204,11 @@ pushGitBranch srcConn branch repo (PushGitBranchOpts setRoot _syncMode) = runExc
             -- Push our changes to the repo
             gitIn remotePath ["push", "--quiet", url]
             pure True
+
+-- | Make a clean copy of the connected codebase into the provided path.
+copyCodebase :: MonadIO m => Connection -> CodebasePath -> m ()
+copyCodebase srcConn destPath = runDB srcConn $ do
+  -- remove any existing codebase at the destination location.
+  removePathForcibly (makeCodebaseDirPath destPath)
```

Comment:
Would `destPath` ever be non-empty?

Reply:
Hrmm, I can't think of any.

In the case that it is empty, it will look make a codebase path relative to the CWD (i.e. `./.unison/v2`), which would be the behaviour I think I'd expect by providing an empty destination.

I double-checked and it won't automatically delete `/` or something 😋 ;

Thanks for the prompt on this though, always good to double-check things like this; wouldn't want to run into the Steam Root situation;

https://github.com/ValveSoftware/steam-for-linux/issues/3671

```
>>> import System.FilePath
>>> "" </> ".unison" </> "v2"
".unison/v2"
```

Reply:
I meant - aren't we only calling (or only supposed to call) `copyCodebase` with a `destPath` that points to an empty directory?

Or might we intentionally call `copyCodebase`, passing a`destPath` that corresponds to a non-empty directory? (And if so, it's ok to remove those files?)